client/app/lib/app_helpers.coffee

```diff
@@ -1,4 +1,4 @@
-(() ->
+( ->
     # IIFE to avoid collisions with other variables
```

Comment:
I'm not really convinced this file is useful (and btw, using `do` for IIFE is more coffee-script compliant, nope? (but this is just to let a note here, it works anyway :grin:)